mcs/build/common/SR.cs

```diff
@@ -46,7 +46,18 @@ internal static string Format(string resourceFormat, object p1, object p2, objec
 		return string.Format (CultureInfo.InvariantCulture, resourceFormat, p1, p2, p3);
 	}
 
-	internal static string GetResourceString (string str) => str;
+	internal static string GetResourceString (string str)
+	{
+		foreach (var field in typeof (SR).GetFields ())
```

Comment:
Why do we need it? (we don't want reflection code in mscorlib)

Reply:
@marek-safar agree. But what should I do with https://github.com/dotnet/corefx/blob/43d07018fd0c4176b20481e958bcd1ccadacc23a/src/Common/src/CoreLib/System/Text/Encoding.cs#L450 - it wants to get a proper name for encoding based on Resources

Reply:
I can make a switch for this case